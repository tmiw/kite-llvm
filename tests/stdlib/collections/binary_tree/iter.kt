t = make System.collections.binary_tree();
t|set("apple", "mac");
t|set("ibm", "pc");
t|set("sun", "unix");

t|reset;
while (t|next)
[
    i = t|cur;
    i[0]|print;
    i[1]|print;
];