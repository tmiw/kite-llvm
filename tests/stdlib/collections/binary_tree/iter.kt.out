apple
mac
ibm
pc
sun
unix
